src/sentry/demo/data_population.py

```diff
@@ -466,20 +507,20 @@ def fix_measurements(event_json):
         measurements.update(measurement_markers)
 
 
-def update_context(event, trace=None):
+def update_context(event, trace=None, platform=None):
+    mobile = platform in mobile_platforms
     context = event["contexts"]
     # delete device since we aren't mocking it (yet)
-    if "device" in context:
+    if "device" in context and not mobile:
```

Comment:
@roggenkemper aren't we going to overwrite the `device` anyways? Why do we need this change?

Reply:
I think I added that as a safety measure before I had finished the updating for mobile. not needed anymore.